codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/customization.config

```diff
@@ -30,5 +30,8 @@
             ]
         }
     },
-    "underscoresInNameBehavior": "ALLOW"
-}
\ No newline at end of file
+    "underscoresInNameBehavior": "ALLOW",
+    "enabledTraitValidations": {
+        "RequiredTrait": true
+    }
```

Comment:
Should we just do `"requiredTraitEnabled": true`? We have to maintain backwards-compatibility on this file for internal customers, and I'm not sure if we're going to add additional traits to this map in the future.

Reply:
Making change.